.github/workflows/stale.yaml

```diff
@@ -0,0 +1,22 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          # Issues
+          stale-issue-message: 'This issue is stale because it has been open 180 days with no activity. Remove stale label or comment or this will be closed in 7 days.'
+          close-issue-message: 'This issue was closed because it has been stalled for 7 days with no activity. Feel free to reopen if still applicable.'
+          days-before-issue-stale: 180
+          days-before-issue-close: 7
```

Comment:
I want to keep good-first-issue labeled issue out of this rule, wdyt?

Reply:
Exempted `good first issue` and `kind/bug`.